src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      if (b <= 0) {
+        return 'ERROR';
+      }
+
+      return a / b;
```

Comment:
[CODE KNOWLEDGE] - It's better to throw an error or return `null` when dividing by zero or a negative number, as returning a string 'ERROR' can be confusing when the rest of the methods return numbers. Also, consider using the arithmetic operators with the assignment operator here as well.